app/assets/stylesheets/common/main.scss

```diff
@@ -20,6 +20,24 @@ body {
     }
 }
 
+.items{
+    height: 85%;
+    table{
+        thead{
+            th{
+                font-size:13px;
+                font-weight: bold;
+                background-color: grey;
+            }
+        }
+        tbody{
+            td{
+            font-size:12px;
+            }
+        }
+    }
+}
+
```

Comment:
These classes look like they are specific to the "my tickets" panel. This should probably be moved to "app/assets/stylesheets/panel/panel.scss" nested under a new "&.user" class.